.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
+}
```

Comment:
I'd prefer if personal config files were kept out of the primary source if possible